recipes/cassiopee/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: cassiopee
+  version: "1.0.5"
+
+build:
+  number: 0
+
+source:
+  fn: 1.0.5.tar.gz
+  url: https://github.com/osallou/cassiopee-c/archive/1.0.5.tar.gz
+  md5: 38f07e5d2ee2a55679281bce73f9993e
+
+requirements:
+  build:
+    - glog
+    - cppunit
+    - boost
+    - autoconf
+    - gcc   # [not osx]
+    - llvm  # [osx]
+    - cmake
+
+  run:
+    - glog
+    - libgcc  # [not osx]
+    - boost
+
+test:
+  commands:
+    - cassiopee -h
+
+about:
+  home: https://github.com/osallou/cassiopee-c
+  license: GPL-3+
```

Comment:
GPL states that you need to distribute the license file, so if you can please use license_file as well.